packages/owl-bot/src/core.ts

```diff
@@ -160,6 +160,14 @@ export async function triggerPostProcessBuild(
     logger.error(`triggerPostProcessBuild: ${err.message}`, {
       stack: err.stack,
     });
+    if (err instanceof TimeoutError) {
```

Comment:
I'd rather see `buildFailureFrom()` get smarter:

```ts
function buildFailureFrom(error: any, detailsUrl: string): BuildResponse {
  if (typeof error.name === 'string' && typeof error.message === 'string) {
    return {
      conclusion: 'failure',
      summary: error.name,
      text: error.message,
      detailsUrl
    };
 } else {
...
}
```

That way, we can see something useful for any kind of exception.

P.S.  I intended to propose this change in my first review ,but I must have clicked the wrong button or something and it got dropped.

Reply:
Done